.vscode/tasks.json

```
@@ -0,0 +1,47 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build",
+            "command": "dotnet",
+            "type": "shell",
+            "args": [
+                "build",
+                // Ask dotnet build to generate full paths for file names.
+                "/property:GenerateFullPaths=true",
+                // Do not generate summary otherwise it leads to duplicate errors in Problems panel
+                "/consoleloggerparameters:NoSummary"
+            ],
+            "group": "build",
+            "presentation": {
+                "reveal": "silent"
+            },
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": ".NET Core Test with debugger",
+            "type": "process",
+            "isBackground": true,
+            "command": "dotnet",
+            "args": [ "test", "--filter", "FullyQualifiedName=LfMerge.Core.Tests.Lcm.DataConverters.ConvertLcmToMongoCustomFieldTests.GetCustomFieldForThisCmObject_ShouldGetMultiListRef" ],
```

remove unnecessary filter
```suggestion
            "args": [ "test" ],
```